lsp-mode.el

```diff
@@ -6441,13 +6461,17 @@ WORKSPACE is the active workspace."
                      ((equal method "client/registerCapability")
                       (mapc #'lsp--server-register-capability
                             (lsp:registration-params-registrations params))
+                      (lsp-unconfig-buffer)
+                      (lsp-configure-buffer)
                       nil)
                      ((equal method "window/showMessageRequest")
                       (let ((choice (lsp--window-log-message-request params)))
                         `(:title ,choice)))
                      ((equal method "client/unregisterCapability")
                       (mapc #'lsp--server-unregister-capability
                             (lsp:unregistration-params-unregisterations params))
+                      (lsp-unconfig-buffer)
```

Comment:
this should be called in all buffers of the workspace.

Reply:
Done